src/Provider/AbstractProvider.php

```diff
@@ -271,13 +272,13 @@ public function getAuthorizationUrl(array $options = [])
         ];
 
         if (is_array($options['scope'])) {
-            $options['scope'] = implode(static::SCOPE_SEPARATOR, $options['scope']);
+            $options['scope'] = join(static::SCOPE_SEPARATOR, $options['scope']);
```

Comment:
Because the alias only exists for people that are confused, and we are not confused.